src/Tribe/Views/V2/View.php

```diff
@@ -1039,9 +1046,25 @@ protected function setup_template_vars() {
 				'month_and_year' => tribe_get_date_option( 'monthAndYearFormat', 'F Y' ),
 			],
 			'messages'          => $this->get_messages( $events ),
+			'breadcrumb_header' => [],
 			'start_of_week'     => get_option( 'start_of_week', 0 ),
 		];
 
+		/**
```

Comment:
:no_entry_sign: Missing short description in doc comment

<sub>hash: 5f30906b5516a770fed31d0919065621</sub>